### PR TITLE
Syslog integration CEF fields improvements

### PR DESCRIPTION
**Pull request by yrro**
## Description

* Fixes the ordering of fields to match the CEF standard
* ~~Changes the CEF signature field to indicate the ID of the policy that was violated~~

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Waiting for CI run. I could add a test to validate the order of CEF fields?

https://github.com/stackrox/stackrox/issues/5400

### CODE REVIEW

**Comment by yrro** [comment]
/retest

The tests are failing with errors like `RROR: Timed out waiting for images after 3600 seconds`. If that's caused by my changes then I can't see where it says that...

**Comment by porridge** [comment]
@yrro looks like there was a problem with the build jobs. Would you mind rebasing your work to resolve the conflict? It should also re-trigger CI and hopefully it will work this time.

File: pkg/notifiers/syslog/syslog.go [hunk 1]
```diff
@@ -188,11 +188,11 @@ func alertToCEF(alert *storage.Alert, notifier *storage.Notifier) string {
 
 	severity := alertToCEFSeverityMap[alert.GetPolicy().GetSeverity()]
 
-	return getCEFHeaderWithExtension("Alert", alert.GetPolicy().GetName(), severity, makeExtensionFromPairs(extensionList))
+	return getCEFHeaderWithExtension(fmt.Sprintf("Policy.{%s}", alert.GetPolicy().GetId()), alert.GetPolicy().GetName(), severity, makeExtensionFromPairs(extensionList))
```
Comment (yrro) [inline pkg/notifiers/syslog/syslog.go:191]: I'm not insisting that the policy ID be exposed here; it just seemed like a nice enhancement. But I'm totally happy to drop this part of you don't agree.

**Comment by yrro** [comment]
No luck yet, @porridge

**Comment by porridge** [comment]
Right, apparently skipping of the pushes is a known issue and we're discussing internally how to solve this.

In the meantime, I raised https://github.com/stackrox/stackrox/pull/6234 to trigger CI. Since it's the same commit, looks like that might be enough for the green to light up in this PR as well.

**Comment by porridge** [comment]
Can someone from @stackrox/core-workflows please review the change?

**Comment by porridge** [comment]
IIRC `ci/prow/gke-postgres-qa-e2e-tests` is now gone, so this PR is good to go from CI standpoint.

**Comment by Maddosaurus** [comment]
@md2119 @theencee @ksurabhi91 As the CI is happy, could we get a review from you folks?

**Comment by theencee** [comment]
@yrro  unfortunately I'm not able to update this PR so I had to create a new one. I'll give you credit for the change in the PR and code as recompense. The new PR: https://github.com/stackrox/stackrox/pull/7527

**Comment by yrro** [comment]
No problem--thanks for taking this forward. I'll close this PR.

**Pull request closed without merging**